### DISCUSSION TITLE
List all dashboards using a specific data source

### DISCUSSION THREAD

**Discussion by pmaus22** [grafana/grafana#73041] [Data Sources]
**What would you like to be added:**
I would like to see all dashboards that use a data source when editing my data sources.

**Why is this needed:**
This would make it easy to see which data sources are outdated/not nessecary. As of now, I have to check all dashboards and panels (!) to see if a specific data source is still in use.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82548. Feel free to continue the discussion around this there. Thank you!
